config/auth.php

```diff
@@ -23,4 +23,126 @@
         ],
     ],
 
+    'oauth' => [
+        // Default providers
+        'facebook' => [
+            'enabled' => env('OAUTH_FACEBOOK_ENABLED'),
+            'icon' => 'tabler-brand-facebook',
+            'color' => \Filament\Support\Colors\Color::hex('#1877f2'),
```

Comment:
Import this at the top!

```suggestion
            'color' => Color::hex('#1877f2'),
```